agent/charms/testflinger-agent-host-charm/src/tf-cmd-scripts/tf-allocate

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+AGENT=$(echo $agent_id | sed 's/-\([0-9]*\)$/\1/g')
+PROVISION_TYPE="$provision_type"
+
+if [ -f /srv/testflinger-agent/$AGENT/testflinger-agent.conf ]; then
+        . /srv/testflinger-agent/$AGENT/env/bin/activate && PYTHONUNBUFFERED=1 testflinger-device-connector $PROVISION_TYPE allocate -c /srv/testflinger-agent/$AGENT/default.yaml testflinger.json
```

Comment:
These scripts were changed a while back to handle the transition to monorepo and putting the config files under `/srv/testflinger-agent/$AGENT`. So we can make them much simpler now, and just keep the first part that runs if that config file exists, and drop the conditional.

Reply:
Thanks! It's been updated.